src/Input.js

```diff
@@ -43,6 +44,7 @@ class Input extends React.Component {
     } = this.props;
 
     const checkInput = ['radio', 'checkbox'].indexOf(type) > -1;
+    const isNotaNumber = new RegExp('(?!^\\d+$)^.+$');
```

Comment:
`/\D/g` would work and is more clear. `new RegExp('\\D','g')` if you want to use `new RegExp`

Reply:
Yes , this is much better. Sorry, i forgot this global flag . Thank you for taking your time for the review. I'll fix it right away.